src/discord/events/message/registerTeam.ts

```diff
@@ -0,0 +1,214 @@
+import Discord from 'discord.js';
+
+import { Config } from '../../../entities/config';
+import logger from '../../../logger';
+import { colors } from '../../constants';
+import { Team } from '../../../entities/team';
+
+class TeamInfo {
+  idArr: string[] = [];
+
+  name = '';
+
+  description = '';
+
+  table = 0;
+
+  user = '';
+
+  constructor(i: string) {
+    this.user = i;
+  }
+
+  addMembers(mem: string[]): void {
+    this.idArr = mem;
+  }
+
+  addUser(): void {
+    this.idArr.push(this.user);
+  }
+
+  addName(n: string): void {
+    this.name = n;
+  }
+
+  addDescription(d: string): void {
+    this.description = d;
+  }
+
+  addTable(t: number): void {
+    this.table = t;
+  }
+}
+
+let registrationInProgress = false;
+let decrement = false;
+let findName: Team;
+let findTable: Team;
+
+const promptArr = [
+  'Who are you hacking with? You have 5 minutes to send a reply. If you are hacking alone, simply write `none`!\n**Be sure to input their Discord ID in a comma-separated list e.g. 111222333444555,666777888999000,...**',
+  "What's your team/app name? You have 2 minutes to answer.",
```

Comment:
Let's clarify this; we've had questions in the past about whether we needed either/or or if we needed two different values
```suggestion
  "What's your team or app name?",
```